### ISSUE TITLE
"File name too long" error

### ISSUE THREAD

**Issue by joshmarinacci** [bwasty/gltf-viewer#26]
I'm trying to load a simple file of a moon.  When I load it with gltf-viewer I get this error: 

```
Jesses-iMac:temp josh$ ./gltf-viewer -vvv moon.gltf
[DEBUG] Renderer     : AMD Radeon HD 6770M OpenGL Engine
[DEBUG] Vendor       : ATI Technologies Inc.
[DEBUG] Version      : 4.1 ATI-1.60.26
[DEBUG] GLSL         : 4.10
[DEBUG] Profile      : Core
[DEBUG] Context Flags: Debug: false, Forward Compatible: true
[DEBUG] Extensions   : GL_ARB_blend_func_extended, GL_ARB_draw_buffers_blend, GL_ARB_draw_indirect, GL_ARB_ES2_compatibility, GL_ARB_explicit_attrib_location, GL_ARB_gpu_shader5, GL_ARB_instanced_arrays, GL_ARB_internalformat_query, GL_ARB_occlusion_query2, GL_ARB_sample_shading, GL_ARB_sampler_objects, GL_ARB_separate_shader_objects, GL_ARB_shader_bit_encoding, GL_ARB_shader_subroutine, GL_ARB_shading_language_include, GL_ARB_tessellation_shader, GL_ARB_texture_buffer_object_rgb32, GL_ARB_texture_cube_map_array, GL_ARB_texture_gather, GL_ARB_texture_query_lod, GL_ARB_texture_rgb10_a2ui, GL_ARB_texture_storage, GL_ARB_texture_swizzle, GL_ARB_timer_query, GL_ARB_transform_feedback2, GL_ARB_transform_feedback3, GL_ARB_vertex_type_2_10_10_10_rev, GL_ARB_viewport_array, GL_EXT_debug_label, GL_EXT_debug_marker, GL_EXT_texture_compression_s3tc, GL_EXT_texture_filter_anisotropic, GL_EXT_texture_mirror_clamp, GL_EXT_texture_sRGB_decode, GL_APPLE_client_storage, GL_APPLE_container_object_shareable, GL_APPLE_flush_render, GL_APPLE_object_purgeable, GL_APPLE_rgb_422, GL_APPLE_row_bytes, GL_APPLE_texture_range, GL_ATI_texture_mirror_once, GL_NV_texture_barrier
[INFO] Imported glTF in         630 µs
[DEBUG] Found no TANGENTS for primitive 0 of mesh 0 (tangent calculation not implemented yet)
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 63, message: "File name too long" } }', src/libcore/result.rs:916:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

This file can be corrected loaded in this online GLTF viewer https://gltf-viewer.donmccurdy.com/

Github won't let me attach a gltf file so I zipped it up first.

[moon.gltf.zip](https://github.com/bwasty/gltf-viewer/files/1794628/moon.gltf.zip)

**Comment by bwasty** [comment]
The error is a bit misleading - the problem is that data URIs are not yet handled for images (the `gltf` crate used to do that, but it was dropped in 0.8 I believe).
Maybe I'll fix that while upgrading to gltf 0.10.1 in the next few days.

**Comment by joshmarinacci** [comment]
Yeah, I figured the real error was something else.  I look forward to the update. thx.
